api/database/migrations/2022_12_09_230812_drop_language_ability_users.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Database\Helpers\ApiEnums;
+use Illuminate\Support\Facades\DB;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        $users = DB::table('users')->get();
+        foreach($users as $user) {
+            $languageAbility = $user->language_ability;
+            $userId = $user->id;
+            // before dropping the field language_ability, execute a data change provided it is non-null
+            if ($languageAbility !== null) {
+
+                // only override looking_for_english if it is null
+                if ($user->looking_for_english === null) {
+                    // play it safe, ENGLISH and BILINGUAL map to true
+                    if ($languageAbility == ApiEnums::LANGUAGE_ABILITY_ENGLISH | $languageAbility == ApiEnums::LANGUAGE_ABILITY_BILINGUAL) {
+                        DB::table('users')->where('id', $userId)->update(['looking_for_english' => true]);
+                    }
+                    else {
+                        DB::table('users')->where('id', $userId)->update(['looking_for_english' => false]);
+                    }
```

Comment:
At this point, I think it makes sense to skip the if statement and just update the DB with a variable.
```suggestion
                    $lookingForEnglish = ($languageAbility == ApiEnums::LANGUAGE_ABILITY_ENGLISH || $languageAbility == ApiEnums::LANGUAGE_ABILITY_BILINGUAL);
                    DB::table('users')->where('id', $userId)->update(['looking_for_english' => $lookingForEnglish]);
```
ps. You probably want to use `||` instead of `|`. See [difference between bitwise and logical operators](https://stackoverflow.com/questions/2376348/difference-between-and-in-php)